models/template.go

```diff
@@ -35,5 +35,6 @@ type Template struct {
 	Description string             `bson:"description"`
 	CreatedAt   time.Time          `bson:"created_at"`
 	Default     bool               `bson:"default,omitempty"`
+	Username    string             `bson:"username"`
```

Comment:
I'm undecided on the data-structure we want to go with yet. We obviously don't want a template to belong to one person via their username. At the least, it would be an ID.

Saying that, I don't know if we want to do some sort of joins-table-esque type deal between users and templates, or if we just straight up want to duplicate the template per use.

Considerations:
- A user may partially fill out a template and won't want it to completely change if the author made amends to it half way through completing it
- A user may want to use the updated version of the template next time they fill it out. Should they perhaps have an option to be able to use the same as last time, too? Maybe take this even a step further and you cannot update templates after creating them, but you can duplicate them? This could solve some problems.
- A user may want to amend the template themselves (is this functionality we even want to offer?)